### PR TITLE
Add samples for Core and ORM features

### PR DESCRIPTION
**Pull request by naveen-ram**

### CODE REVIEW

File: samples/Core.py [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    stmt = users.update(). where(users.c.name == 'jack').values(name='ed')
+    conn.execute(stmt)
+
+    # Deletion
+    conn.execute(users.delete().where(users.c.name > 'h'))
+
+    result = conn.execute(select([users]))
+
+    for row in result:
+        print(row)
```
Comment (kadler) [inline samples/Core.py:96]: Missing end of file newline

File: samples/Core.py [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+              )
+           )
+
+    print(conn.execute(s).fetchall())
+
+    # Textual SQL
+
+    s = text("SELECT users.fullname || ', ' || addresses.email_address AS"
+             " title FROM users, addresses WHERE users.id = addresses.user_id "
+             "AND users.name BETWEEN :x AND :y "
```
Comment (kadler) [inline samples/Core.py:80]: I don't think we support named parameter markers, do we? I thought pyodbc only supported qmark parameter markers.

Comment (naveen-ram) [reply samples/Core.py:80]: They are converted into qmarks by pyodbc

Comment (kadler) [reply samples/Core.py]: Oh, cool

**Pull request merged**